src/components/text-input/text-input.stories.tsx

```diff
@@ -56,9 +56,9 @@ export const withIconAndComplexHelp = () => (
     label="Hand size"
     placeholder="Type here..."
     help={
-      <Paragraph paragraphSize="small" marginTop="8px" color="grey.light.1">
+      <Paragraph paragraphSize="small" marginTop="8px" color="neutral.2">
         This element is very{' '}
-        <Anchor fontSize="8px" href="/" color="primary.normal" fontWeight="bold">
+        <Anchor fontSize="8px" href="/" color="primary.4" fontWeight="bold">
```

Comment:
```suggestion
        <Anchor fontSize="8px" href="/" color="accent.6" fontWeight="bold">
```
accent colorokat használjunk az interakcióknál